ios/Podfile

```diff
@@ -0,0 +1,120 @@
+require_relative '../packages/react-native-unimodules/cocoapods.rb'
+
+source 'https://cdn.cocoapods.org/'
+platform :ios, '10.0'
+
+install! 'cocoapods',
+         :generate_multiple_pod_projects => true,
+         :incremental_installation => true
+
+inhibit_all_warnings!
+
+target 'Exponent' do
+  # Expo Client dependencies
+  pod 'CocoaLumberjack', '~> 3.5.3'
+  pod 'Fabric', '~> 1.10.2'
+
+  # Client & ExpoKit dependencies
+  pod 'Crashlytics', '~> 3.13.4'
+
+  # ExpoKit dependencies
+  pod 'GoogleMaps', '~> 3.3'
+  pod 'Google-Maps-iOS-Utils', '~> 2.1.0'
+  pod 'lottie-ios', '~> 2.5.0'
+  pod 'JKBigInteger2', '0.0.5'
+
+  # Unimodules
+  use_unimodules!({
+    modules_paths: ['../packages'],
+    exclude: [
+      'expo-module-template',
+      'expo-bluetooth',
+      'expo-in-app-purchases',
+      'expo-payments-stripe',
+      'expo-updates',
+    ],
+    flags: {
+      :inhibit_warnings => false
+    }
+  })
+
+  # Unversioned React Native
+  pod 'React',
+    :path => "../react-native-lab/react-native",
+    :subspecs => [
+      "Core",
+      "ART",
+      "RCTActionSheet",
+      "RCTAnimation",
+      "RCTCameraRoll",
+      "RCTGeolocation",
+      "RCTImage",
+      "RCTNetwork",
+      "RCTText",
+      "RCTVibration",
+      "RCTWebSocket",
+      "DevSupport",
+      "CxxBridge"
+    ]
+  pod 'yoga',
+    :path => "../react-native-lab/react-native/ReactCommon/yoga"
+  pod 'DoubleConversion',
+    :podspec => "../react-native-lab/react-native/third-party-podspecs/DoubleConversion.podspec"
+  pod 'Folly',
+    :podspec => "../react-native-lab/react-native/third-party-podspecs/Folly.podspec"
+  pod 'glog',
+    :podspec => "../react-native-lab/react-native/third-party-podspecs/glog.podspec"
+
+  # Evaluate all files matching `versioned-react-native/ABI*/dependencies.rb` glob pattern
+  # and install ReactABIXX_0_0 pods with all versioned unimodules.
+  use_versioned_abis
+
+  # Build React Native with RCT_DEV enabled and RCT_ENABLE_INSPECTOR and
+  # RCT_ENABLE_PACKAGER_CONNECTION disabled
+  post_install do |installer|
+    # Disabled as of CocoaPods 1.8.0.beta1 since pods_project seems to be nil
+    # installer.pods_project.main_group.tab_width = '2';
+    # installer.pods_project.main_group.indent_width = '2';
+
+    installer.target_installation_results.pod_target_installation_results
+      .each do |pod_name, target_installation_result|
+
+      # Run postinstalls actions for versioned dependencies.
+      # These actions are specified in `versioned-react-native/ABI*/postinstalls.rb` files.
+      run_versioned_postinstalls(pod_name, target_installation_result)
+
+      target_installation_result.native_target.build_configurations.each do |config|
+        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '10.0'
+      end
+
+      # Can't specify this in the React podspec because we need to use those podspecs for detached
+      # projects which don't reference ExponentCPP.
+      if pod_name.start_with?('React')
```

Comment:
Maybe, if we're already changing this code, we would like to move this to dedicated Podfile parts? (the only `React*` pods in this Podfile are `React`, `ReactABI33_0_0`,…) And this logic of
```rb
if pod_name.start_with?('React')
  # do something
end

if pod_name === 'React'
  # do something more
end
```
looks like something that could have been done better, I think, and moving all the versioned code to versioned files would make it less error-prone

Reply:
Actually we can remove this `if` completely... 
- `IPHONEOS_DEPLOYMENT_TARGET` is already set for all deps by the previous build_configurations loop
- the client compiles without setting `HEADER_SEARCH_PATHS` to inherited
Are you fine with that? Do you remember the reason why we set header search paths there?

Reply:
If the client compiles (make sure to delete DerivedData and test again) then I think we don't need to set the HEADER_SEARCH_PATHS. CocoaPods or Xcode's behavior may have changed.